asApp/src/save_restore.c

```diff
@@ -2565,7 +2566,7 @@ STATIC int remove_data_set(char *filename)
 
     msg.operation = op_Remove;
     if ((filename == NULL) || (strlen(filename) < 1) || (strlen(filename) >= OP_MSG_FILENAME_SIZE - 1)) {
-        printf("remove_data_set: bad filename\n");
+        ERRLOG("bad filename\n");
         return (-1);
```

Comment:
While we're at it, maybe change this to `ERROR` as well?

Reply:
Yeah, This also bothered me. We should have another PR for that.